src/SFA.DAS.EmployerUsers.Users.Database/StoredProcedures/SearchUsers.sql

```diff
@@ -18,7 +18,10 @@ AS
 		IsLocked
 	INTO #resultSet
 	FROM dbo.[User]
-	WHERE UPPER(FirstName) LIKE @criteria OR UPPER(LastName) LIKE @criteria OR UPPER(Email) LIKE @criteria
+	WHERE UPPER(FirstName) LIKE @criteria 
+	OR UPPER(LastName) LIKE @criteria 
+	OR UPPER(FirstName) + ' ' + UPPER(LastName) LIKE @criteria
+	OR UPPER(Email) LIKE @criteria
```

Comment:
The 'UPPER' can be removed - its an operation that isn't needed and will have cost on the query in the Where clause.